src/Moryx.ControlSystem/VisualInstructions/VisualInstructorExtensions.cs

```diff
@@ -96,7 +96,8 @@ public static long Execute(this IVisualInstructor instructor, string title, Acti
             {
                 Title = title,
                 Instructions = instructions,
-                PossibleResults = results
+                PossibleResults = results,
```

Comment:
Did you check this warning? Looks good, otherwise.

Reply:
Yeah, the obsolote warning. But for now we still need to provide the value